mods/ts/sequences/misc.yaml

```diff
@@ -609,7 +609,7 @@ tracks16:
 		Offset: 0, -13, 42
 		ZRamp: 1
 		UseTilesetExtension: true
-		ShadowStart: 1
+		#ShadowStart: 1
```

Comment:
This part of the tunnel shadow wasn't even used in the original game (at least on the version I tested, it might have been fixed or broken in a patch).